src/ElectricPokemon.java

```diff
@@ -0,0 +1,123 @@
+import java.util.List;
+import java.util.Arrays;
+
+public class ElectricPokemon extends Pokemon{
+    private static final String type = "electric";
```

Comment:
Dit is inderdaad een goede `private static final` constante, in plaats van bijvoorbeeld een "magic string". De type in de super klasse heeft een getter, deze geef je in de constructor netjes door aan de super.